### ISSUE TITLE
Request for reopening #110

### ISSUE THREAD

**Issue by jonatan-ivanov** [lkishalmi/gradle-gatling-plugin#111]
Could you please check if you can repro this potential bug described in #110?
I was able to repro it with a very little modification on the `sample-project`: https://github.com/jonatan-ivanov/gradle-gatling-plugin/commit/d195e3480c214e6f9815fe926813be87731ac6eb

It seems if there is no resources folder or that folder is empty, the log level is `DEBUG` (or lower).
Could you please try it on your side to see what happens? You can use my change (https://github.com/jonatan-ivanov/gradle-gatling-plugin/commit/d195e3480c214e6f9815fe926813be87731ac6eb) or delete the resources folder and fix `SampleSimulation`.

Please:
- Close this issue if you think this does not worth the effort to investigate/fix
- Reopen #110 if you were able to repro it and close this one
- Let me know if you were not able to repro it but please keep this issue open so that I can try to investigate this further

**Comment by eshepelyuk** [comment]
I took a look at the description and there's no description how I can reproduce the bug you've submitted.

I'd generated sample project as described in README - and as soon as I change `logLevel` in `build.gradle` - the change is applid to `Galting` logger.

So you may do following

1. Generate a sample project as described in README.
1. with no changes run `gatingRun` and caprutre the output
1. change logLevel in `build.gradle` and capture the output
1. attach a link to sample project and both captured output to the bug, so I can try to reproduce

**Comment by jonatan-ivanov** [comment]
TL;DR: Delete the resources folder, please. :)

@eshepelyuk Let me clarify it a little and provide the repro steps (they are basically in the opening comment but let me give a step-by-step):

1. Check out https://github.com/jonatan-ivanov/gradle-gatling-plugin/commit/d195e3480c214e6f9815fe926813be87731ac6eb or follow your own steps above but **delete the `resources` folder** in your second step and fix `SampleSimulation` (see: https://github.com/jonatan-ivanov/gradle-gatling-plugin/commit/d195e3480c214e6f9815fe926813be87731ac6eb)
2. Run `gatlingRun` and watch the output
3. The log level should be `WARN` but it seems it is `DEBUG` if there is no `resources` folder or that folder is empty

>You should be understanding that all those steps I'd already tried during the investigation of #110 and I didn't experience any bugs.

I understand you followed those steps, what I'm saying is following those does not repro the issue. You need to remove the `resources` folder or the folder needs to be empty (see: https://github.com/jonatan-ivanov/gradle-gatling-plugin/commit/d195e3480c214e6f9815fe926813be87731ac6eb)

I tried it again and followed your steps with one modification: after I generated the sample project (as the Readme said), I  deleted the resources folder and fixed the `SampleSimulation` (see: https://github.com/jonatan-ivanov/gradle-gatling-plugin/commit/d195e3480c214e6f9815fe926813be87731ac6eb): `val feeder = Array(Map("searchCriterion" -> "foo", "searchComputerName" -> "bar")).random`
After this, when I ran `gatlingRun` I saw a lot of `DEBUG` events on the output while I think I only should have seen `WARN` events.

**Comment by eshepelyuk** [comment]
@jonatan-ivanov we are in process on major and breaking infrastructure changes currently. Sorry, currntly concentrated on those. Hope in couple of weeks we can get back to the issue.

**Comment by eshepelyuk** [comment]
This should be addressed in upcoming releases of https://github.com/gatling/gatling-gradle-plugin.

Closing the issue.

**Comment by jonatan-ivanov** [comment]
@eshepelyuk May I ask what is the root cause and what is the proposed fix?

**Comment by ScottG489** [comment]
Moving conversation here from my original issue (#115) since it seems to be a duplicate.

Hey @eshepelyuk. It might be a good idea to keep this ticket open until the fix has been released.

**Comment by jonatan-ivanov** [comment]
This issue seems to be the same as these newer ones in the `gatling/gatling` repo:
- https://github.com/gatling/gatling/issues/3951
- https://github.com/gatling/gatling/issues/3946

**Comment by jonatan-ivanov** [comment]
@eshepelyuk I agree with @ScottG489: it would be nice to keep this ticket open until the fix has been released and verified.
